examples/tv-casting-app/APIs.md

```diff
@@ -239,41 +274,50 @@ public static void demoDiscovery()
 On iOS, register the listeners by calling `addObserver` on the `NotificationCenter` with the appropriate selector, and then call `start` on the `sharedInstance` of `MTRCastingPlayerDiscovery`.
 ```objectivec
 func startDiscovery() {
-        NotificationCenter.default.addObserver(self, selector: #selector(self.addDiscoveredCastingPlayers), name: NSNotification.Name.didAddCastingPlayers, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(self.removeDiscoveredCastingPlayers), name: NSNotification.Name.didRemoveCastingPlayers, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(self.updateDiscoveredCastingPlayers), name: NSNotification.Name.didUpdateCastingPlayers, object: nil)
-
-        MTRCastingPlayerDiscovery.sharedInstance().start()
-        self.discoveryRequestStatus = true
-    }
+    NotificationCenter.default.addObserver(
+    self,
+    selector: #selector(self.addDiscoveredCastingPlayers),
+    name: NSNotification.Name.didAddCastingPlayers,
+    object: nil)
+    NotificationCenter.default.addObserver(
+    self,
+    selector: #selector(self.removeDiscoveredCastingPlayers),
+    name: NSNotification.Name.didRemoveCastingPlayers,
+    object: nil)
+    NotificationCenter.default.addObserver(
+    self,
+    selector: #selector(self.updateDiscoveredCastingPlayers),
+    name: NSNotification.Name.didUpdateCastingPlayers,
+    object: nil)
```

Comment:
nit: I thought the single liner NotificationCenter statements were easier to read. You can keep the new formatting if you did it deliberately, just thought I'll call out that it looks odd to me.

Reply:
I don't have a strong opinion here and am fine if you want to revert it.  It just went to a long line that was inconsistent with the line lengths of other samples, requiring left-->right scrolling.